sklearn/model_selection/_split.py

```diff
@@ -601,8 +601,20 @@ class StratifiedKFold(_BaseKFold):
 
     Notes
     -----
-    Train and test sizes may be different in each fold, with a difference of at
-    most ``n_classes``.
+    The implementation is designed to:
+
+    * Generate test sets such that all contain the same distribution of
+      classes, or as close as possible.
+    * Be invariant to class label: relabelling ``y = ["Happy", "Sad"]`` to
```

Comment:
maybe more straightforward to use y=[0, 1]?

Reply:
I think the translation is less clear to the reader if I use [0,1]